Finjector.Web/Extensions/QueryExtensions.cs

```diff
@@ -0,0 +1,12 @@
+using System.Linq.Expressions;
+using Finjector.Core.Domain;
+
+namespace Finjector.Web.Extensions;
+
+public static class QueryExtensions
+{
+    public static Func<string, Expression<Func<Folder, bool>>> GetFolderCondition = (iamId) => (f =>
```

Comment:
Nice

Reply:
took me a while!  I still don't fully understand why i need to do `toQueryable()` sometimes for it to work.  But it does, and all in the db!